extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/theta/SketchAggregatorFactory.java

```diff
@@ -266,6 +267,22 @@ public byte[] getCacheKey()
                      .array();
   }
 
+  @Override
+  public AggregatorFactory substituteCombiningFactory(AggregatorFactory preAggregated)
```

Comment:
does `SketchMergeAggregatorFactory` need to override this with an impl that checks `isInputThetaSketch`?

Reply:
hmm, i suppose it wouldn't hurt, though i can't imagine the field would be the same name and `isInputThetaSketch` be true in query agg but not in projection agg, or the other way, can add just in case since i need to make some other changes